content/en/docs/Installation/linux.md

```diff
@@ -13,11 +13,130 @@ aliases:
+
+Using `apt`:
+
+~~~bash
+apt install ./navidrome_0.XX.X_linux_amd64.deb
+~~~
+
+Using `dpkg`:
+
+Install the package and then resolve the dependancies:
+
+~~~bash
+sudo dpkg -i ./navidrome_0.XX.X_amd64.deb
+sudo apt install -f
+~~~
+
+**Configuration File**: After installation, Navidrome MUST be configured to run. The default path for the configuration file is /var/lib/navidrome/navidrome.toml. Create and edit the file using nano directly.
+
+~~~bash
+sudo nano /var/lib/navidrome/navidrome.toml
+~~~
+
+Add the following line to specify your music library path:
+
+~~~conf
+MusicFolder = "/path/to/your/music/library"
+~~~
+
+For additional configuration options see the [configuration options page](https://www.navidrome.org/docs/usage/configuration-options/).
+
+**Start the Navidrome Service**: Use systemctl to start the Navidrome service.
+
+~~~bash
+sudo systemctl start navidrome
+~~~
+
+**Enable Navidrome to Start on Boot**: To make Navidrome start automatically on system boot, enable the service.
+
+~~~bash
+sudo systemctl enable navidrome
+~~~
+
+**Check Service Status**: Verify that Navidrome is running correctly.
+
+~~~bash
+sudo systemctl status navidrome
+~~~
+
+If everything is set up correctly, Navidrome will be accessible via web browser: http://localhost:4533.
+
+
+
+## Migrate from self-built to .deb Pre-built package
+
+Migrating from a self-built Navidrome instance to the new pre-built .deb package version can streamline updates and maintenance. This guide will walk you through the process of migrating your existing Navidrome setup on Linux to the .deb package version, specifically from the 0.54.1 release.
+
+Before starting the migration, ensure you have:
+
+* **Backup**: Always back up your current Navidrome configuration and music library. This includes the navidrome.toml configuration file and any other custom settings you may have.
+* **System Update**: Make sure your system is up to date.
+
+~~~bash
+sudo apt update
+sudo apt upgrade
+~~~
+
+### Remove Existing Program
+
+First, stop the currently running Navidrome service to prevent any conflicts during the installation of the new package.
+
+~~~bash
+sudo systemctl stop navidrome.service
+~~~
+
+Navigate to the directory where your self-built Navidrome is located and remove the files. Be cautious not to delete your configuration or music library.
+
+~~~bash
+sudo rm -rf /opt/navidrome
+~~~
+
+### Installation
+
+The machine is now clean and ready for installation. Follow the regular [Linux installation instructions](#install-navidrome-using-pre-built-binary) above. Just be sure to place the config file and database in appropriate locations (/var/lib/navidrome by default).
+
+
+### Additional Considerations
+
+* **Permissions**: Ensure that the user running Navidrome has the necessary permissions to access your music library.
```

Comment:
Probably note again here that the user is `navidrome` (the deb/rpm creates a `navidrome` user)